### ISSUE TITLE
Alert icons are showing in places where they shouldn't

### ISSUE THREAD

**Issue by jseppi** [18F/federalist#986]
This is a bug/regression currently in staging, but not production, caused by the fixes to some CSS image paths made by #798. It needs to be fixed before the next deployment to production.

This is in 
<img width="905" alt="screen shot 2017-06-07 at 2 39 01 pm" src="https://user-images.githubusercontent.com/697848/26897760-3ac26066-4b8f-11e7-804e-b8205c099adc.png">

<img width="410" alt="screen shot 2017-06-07 at 2 39 13 pm" src="https://user-images.githubusercontent.com/697848/26897767-3fa6d9d6-4b8f-11e7-8c65-b79772bd56b7.png">

Basically, we were using the `.alert` class in several non-alert contexts. It had CSS to specify some icon paths, but the paths were wrong, so the icons were not displayed. However, after fixing that CSS in #966, now the icons are showing, and it is breaking the intended design.

**Comment by wslack** [comment]
@jseppi #798 was fixed in March. What happened recently to trigger this in staging only?

**Comment by jseppi** [comment]
Sorry, auto-selected the wrong PR! Should be #966. I've edited the root comment.
